lib/pbench/server/api/__init__.py

```diff
@@ -313,14 +312,15 @@ def post(self):
             )
             abort(400, message="Missing required content-length header")
         else:
-            if content_length > _MAX_UPLOAD_DATA_SIZE:
+            max_upload_size = int(eval(app.config["MAX_CONTENT_LENGTH"]))
```

Comment:
Not sure about `eval` here, why do we need it?

Reply:
Max content length value is stored as `100 * 1024 * 1024` in config file

Reply:
Not sure we should support arbitrary strings in the config file.  Let's consider modifying the config file to contain a "humanized" string, and see if that humanize python module can help us read it.

Reply:
I think the `humanize` module is one directional meaning from bit/bytes to human readable format. https://python-humanize.readthedocs.io/en/latest/filesize/, I will write our own small function to convert from human readable to bytes and we can use humanized string in our config file